api-reference/beta/api/group_validateproperties.md

```diff
@@ -0,0 +1,133 @@
+# group: validateProperties
```

Comment:
nit: Would it be useful to describe why one might use this API over the other one? It is used for Group update scenarios so it won't validate mailNickname uniqueness against its own mailNickname.

Reply:
I get that, but that seems like unnecessary detail to me.  I would prefer to just provide the guidance that prior to group updates to displayName or mailNickname, use this API to validate the displayName and mailNickName.